src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objProp = {};
+
+  sourceString.trim()
+    .split(';')
```

Comment:
If you use several array functions in chain it will be better to place them on separate lines.
Not sure if you really need the first `trim`
```suggestion
  sourceString
    .trim()
    .split(';')
```